dataset/dictionary/y-events.json5

```diff
@@ -727,20 +810,23 @@
   {
     en: "Thunder Sojourn",
     ja: "雷痕を求めて",
+    zhCN: "谒索雷痕",
     pronunciationJa: "らいこんをもとめて",
     notes: "v2.0 期間限定イベント",
     tags: [ "event" ],
   },
   {
     en: "Phantom Flow",
     ja: "幻影心流",
+    zhCN: "幻影心流",
     pronunciationJa: "げんえいしんりゅう",
     notes: "v2.0 期間限定イベント",
     tags: [ "event" ],
   },
   {
-    en: "Theater Machanicus: Stage of Wonders",
+    en: "Theater Mechanicus: Stage of Wonders",
```

Comment:
Can you add the redirect setting to [redirect/words.json](https://github.com/xicri/genshin-langdata/blob/main/dataset/redirect/words.json) as well?

The current URL is https://genshin-dictionary.com/ja/theater-machanicus-stage-of-wonders/, so you can configure the redirect by adding the following line:

```json
"theater-machanicus-stage-of-wonders": "theater-mechanicus-stage-of-wonders"
```